File: documentation/user/en/query/requirements/examples/fetching/referenceContentOrder.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityPrimaryKeyInSet(103885),
					entityLocaleEquals(Locale.forLanguageTag("en"))
				),
				require(
					entityFetch(
						attributeContent("name"),
						referenceContent(
							"parameterValues",
							orderBy(
								entityProperty(
									attributeNatural("name", ASC)
								)
							),
							entityFetch(
								attributeContent("name")
							)
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/fetching/referenceContentOrder.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityPrimaryKeyInSet(103885),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en"))
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("name"),
        			ReferenceContent(
        				"parameterValues",
        				OrderBy(
        					EntityProperty(
        						AttributeNatural("name", Asc)
        					)
        				),
        				EntityFetch(
        					AttributeContent("name")
        				)
        			)
        		)
        	)
        )
	)
);